### PR TITLE
Removes some obselete attackby() code [Removes attackby crafting]

### PR DESCRIPTION
**Pull request by Bawhoppen**
title accurate

### CODE REVIEW

**Comment by Cheridan** [comment]
Good shit. Make sure you check all the maps, I'm sure metastation has wire rods laying around because he likes to put every type path on the map

**Comment by PKPenguin321** [comment]
Did you know spears have no tablecrafting recipe

Also I hate this

**Comment by Bawhoppen** [comment]
typo

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -9,6 +9,13 @@
 	var/chem_catalysts[] = list() //like tools but for reagents
 	var/category = CAT_NONE //where it shows up in the crafting UI
 
+/datum/crafting_recipe/stunprod
+	name = "Stunprod"
```
Comment (PKPenguin321) [inline code/modules/crafting/recipes.dm]: Copy paste detected

Comment (Bawhoppen) [inline code/modules/crafting/recipes.dm]: see most recent commit name

**Comment by Firecage** [comment]
You could atleast have put an actual title, or at the very least put what this PR does.

I dislike this, since I bloody well hate tablecrafting(or whatever it is called now). It's less fun and less intuitive.

**Comment by Bawhoppen** [comment]
you can craft literally everywhere now

File: code/modules/crafting/recipes.dm [hunk 2]
```diff
@@ -9,6 +9,13 @@
 	var/chem_catalysts[] = list() //like tools but for reagents
 	var/category = CAT_NONE //where it shows up in the crafting UI
 
+/datum/crafting_recipe/spear
+	name = "Spear"
+	result = /obj/item/weapon/twohanded/spear
+	reqs = list(/obj/item/weapon/restraints/handcuffs/cable = 1,
+				/obj/item/stack/rods = 1)
```
Comment (Firecage) [inline code/modules/crafting/recipes.dm]: While I dislike this PR....you kinda made spears slightly easier to craft. You forgot the glass shard in the recipe. Honk.

Comment (Bawhoppen) [inline code/modules/crafting/recipes.dm]: I knew I was forgetting something

**Comment by Cheridan** [comment]
@Firecage THERE IS NOTHING INTUITIVE AT ALL ABOUT HANDCRAFTING
THE NEXT PERSON TO SAY THAT HANDCRAFTING IS INTUITIVE IS GETTING A BOX OF SCORPIONS MAILED TO THEM

**Comment by Firecage** [comment]
@Cheridan I actually wouldn't mind that...You know my address, right?

**Comment by lzimann** [comment]
crafting is bad and you should feel bad

**Comment by Razharas** [comment]
Personal crafting is very good both code wise and gameplay wise
Only thing it lacks is immersion but it can be fixed with animations more advanced than progress bar

**Comment by Supermichael777** [comment]
Slap crafting is both intuitive and realistic

**Comment by Dorsisdwarf** [comment]
It's not intuitive. 
What you mean by "Intuitive" is "I'm used to it".
Thats not the same thing.
get rod -> Get cables - > click cables in-hand to make ties -> Switch hands -> Click on ties with rod -> Select head -> Click _on the rod_ with head (Opposite way round to last step) -> Batteries maybe

Get Rod -> get cables -> Get head -> Open crafting menu -> Go to weapons -> Click weapon of choice

**Comment by Supermichael777** [comment]
because clickdragging a table to yourself is somehow more intuitive then using zipties to tie things to eachother

**Comment by Razharas** [comment]
We moved away from tables a few weeks ago m8, it is now a button on your screen like inventory
Also you dont need to understand ui to add new recipes
Adding recipe takes 8 lines of code, making a slap-version of it takes 72 lines(as you can judge by the diff count of this pr) and an additional item type that is purely cosmetical

**Comment by Wyzack** [comment]
Our crafting system uses a clunky as fuck menu that sucks to navigate through and can be laggy. Slapcrafting is intuitive, you use the cable to tie things together. Also there is a fucking freeze on and removing something that people have been using forever and that you know is controversial is not a bug fix

**Comment by MMMiracles** [comment]
B-but change is bad! 

I don't care that its more straight forward, less messy, and is easier to maintain, IT'S JUST NOT THE SAME.

**Comment by Bawhoppen** [comment]
you could carry a glass shard and a wire rod in your backpack before and instantly make a spear

**Comment by CosmicScientist** [comment]
>instantly make a spear
Doesn't work like that. You need to pull out three items separately that all take up space in your backpack and then tap the right thing onto the other right thing all the while everyone else is likely noticing your suspicious lack of activity.

I'd probably have a laff in deadchat if that killed me since it's hardly commonplace and not without drawbacks.


From a real standpoint, the old attackby stuff makes sense. From a game standpoint, I struggle to enjoy table crafting due to the sections and the sheer bloat of items in those sections, although I hope it's not like that but I imagine it is since I've not touched it in the game for two months.

**Comment by Dorsisdwarf** [comment]
1) Baww was talking about the _old_ system and stating it as a negative you dip

2) Tablecrafting has been dead for weeks, there's literally a HUD button for crafting

**Comment by Bawhoppen** [comment]
@CosmicScientist with old system you carried a wired rod and a glass shard, you click wired rod with glass shard and you instantly got a spear

**Comment by Dorsisdwarf** [comment]
@Wyzack 
Cast your mind back to the distant far-off days of when you were brand new. Did you think "I bet I can make a stunprod by..." and then intuitively put together the steps? Or did you just memorise the nearly arbitary steps from the wiki?

**Comment by TrustyGun** [comment]
I dont get how slapping items together in a specific, awkward way to make something is more intuitive then getting all the items and using a simple (however laggy) interface to make it

:+1:

**Comment by Anonmare** [comment]
I can slap-craft faster than the craft menu can craft, and that doesn't factor in me having to bring up a huge ass menu and scrolling through it to find what I want.

Leave slap-craft in, let it be something for people to discover to fast craft with.

**Comment by Bawhoppen** [comment]
this fixes being able to instantly make a spear from components in your backpack

**Comment by Anonmare** [comment]
Okay, because apparently someone deleted my comment. Leave slap-crafting in as a faster alternative to manual-crafting. It's faster and it'll be nice for new players to learn and discover. I remember the first time I learnt about being able to make improvised weapons and I thought that was way cool.

Plus not all servers have manual craft so migrant players will try to slap-craft first and I don't think we should punish them for that.

**Comment by xxalpha** [comment]
One of the things I enjoy SS13 for is the IC methods for doing things, such as slapping items together to assemble new items.

So anyone in favor of this doesn't like IC. Menu crafting is just streamlining IC interactions to OOC interfaces.

Might as well print assembled cyborgs instead of each part individually, among other things, for consistency?

**Comment by Bawhoppen** [comment]
actually I just wanted to piss off the powergamers

**Comment by xxalpha** [comment]
What does this have to do with powergaming?

**Comment by xxalpha** [comment]
Is this the birth of a new paprika?

**Comment by Bawhoppen** [comment]
how many times do I have to explain this

YOU CARRY WIRED ROD AND GLASS SHARD IN BACKPACK, USE GLASS SHARD ON WIRED ROD FOR INSTANT SPEAR

**Comment by xxalpha** [comment]
So what?

**Comment by Bawhoppen** [comment]
why is git so broken

**Comment by xxalpha** [comment]
This is a problem with spears, removing an entire portion of the game to grudgecode some people is not acceptable.

Truly the birth of a new paprika.

**Comment by Bawhoppen** [comment]
honestly it isn't a big of deal

**Comment by PKPenguin321** [comment]
>YOU CARRY WIRED ROD AND GLASS SHARD IN BACKPACK, USE GLASS SHARD ON WIRED ROD FOR INSTANT SPEAR

Put a delay on the last step of spear crafting then? It's not hard
In fact it's easier than what you're doing here

**Comment by Bawhoppen** [comment]
nah this is more memey and alienating

**Comment by PKPenguin321** [comment]
Close this pr

**Comment by lzimann** [comment]
you mean you did this pr just to spite some people? just get the fuck out already baw

**Comment by Bawhoppen** [comment]
I did it for the meme

**Comment by Shadowlight213** [comment]
As people have pointed out, menu crafting is extremely ooc.
I know the improve not remove is a meme, but why not actually find a way to improve slap crafting?

**Comment by Bawhoppen** [comment]
go ahead and code datum recipes and we'll all gladly accept it

**Comment by PKPenguin321** [comment]
Reminder that there is literally nothing wrong with slap crafting

**Comment by xxalpha** [comment]
Tell me, if we do this, why not remove cyborg assembly, machine/computer assembly, etc.?

**Comment by Bawhoppen** [comment]
cause thats silly

**Comment by xxalpha** [comment]
It is no different than this.

**Comment by Bawhoppen** [comment]
but it is

**Comment by PKPenguin321** [comment]
Not really why not just let people tablecraft Borg shells???? Because its boring is why

**Comment by ike709** [comment]
slapping together a spear and slapping together a borg are the same process

**Comment by PKPenguin321** [comment]
reminder that there is **l i t e r a l l y** no reason we can't have UI crafting and slapcrafting living side by side

**Comment by Bawhoppen** [comment]
this was just a meme, I never intended it to get merged but xxalpha was so retarded I almost want it to be merged now

but I am the better man so

**Comment by PKPenguin321** [comment]
You redeemed yourself baw I'm proud

**Comment by Incomptinence** [comment]
STAY STRONG RALLY THE POPPERS! 

BANISH THE FACE OF REE!

**Comment by Kromgar** [comment]
>Implying codefreeze is real

**Pull request closed without merging**